rolldown.config.js

```diff
@@ -0,0 +1,49 @@
+// @ts-check
+import { isAbsolute } from 'node:path';
+import wyw from '@wyw-in-js/rollup';
+import pkg from './package.json' with { type: 'json' };
+import { defineConfig } from 'rolldown';
+
+/**
+ * TODO: https://github.com/rolldown/rolldown/issues/3200
+ * @type {readonly import('rolldown').OutputOptions[]}
+ */
+const outputs = [
+  {
+    dir: 'lib',
+    entryFileNames: 'bundle.js',
+    cssEntryFileNames: 'styles.css',
+    format: 'es',
+    sourcemap: true
+  },
+  {
+    dir: 'lib',
+    entryFileNames: 'bundle.cjs',
+    cssEntryFileNames: 'styles.css',
+    format: 'cjs',
+    sourcemap: true
+  }
+];
+
+export default outputs.map((output) =>
+  defineConfig({
+    input: './src/index.ts',
+    output,
```

Comment:
output cannot be an array?

Reply:
It couldn't before, it can be now. (https://github.com/rolldown/rolldown/issues/3200)